### PR TITLE
write liberty file alongside .lef files for macros

### PR DESCRIPTION
**Pull request by oharboe**
Signed-off-by: Øyvind Harboe <oyvind.harboe@zylin.com>

### CODE REVIEW

**Comment by QuantamHD** [comment]
@maliberty

File: flow/Makefile [hunk 1]
```diff
@@ -185,9 +185,12 @@ ifdef BLOCKS
   $(foreach block,$(BLOCKS),$(info Block ${block} needs to be hardened.))
   $(foreach block,$(BLOCKS),$(eval BLOCK_CONFIGS += ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/$(block)/config.mk))
   $(foreach block,$(BLOCKS),$(eval BLOCK_LEFS += ./results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/${block}.lef))
+  $(foreach block,$(BLOCKS),$(eval BLOCK_LIBS += ./results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/${block}.lib))
   $(foreach block,$(BLOCKS),$(eval BLOCK_GDS += ./results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/6_final.gds))
   $(foreach block,$(BLOCKS),$(eval BLOCK_CDL += ./results/$(PLATFORM)/$(DESIGN_NICKNAME)_$(block)/$(FLOW_VARIANT)/6_final.cdl))
   export ADDITIONAL_LEFS += $(BLOCK_LEFS)
+  # FIXME why do we need to use LIB_FILES and not ADDITIONAL_LIBS here?
```
Comment (maliberty) [inline flow/Makefile]: I'm not sure what the issue is here but I don't want to invest the time right now to investigate it fully.

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
I'm not sure what happened with the CI so I restarted the runs

**Comment by maliberty** [comment]
@gadfort I believe that getTechLayer() will return a nullptr for a via inside a pin.  Is this something you can handle?

File: flow/Makefile (hunk 1 above)
Comment (oharboe) [reply flow/Makefile]: Fixed. Using ADDITIONAL_LIBS works.

**Comment by oharboe** [comment]
@maliberty Fixed the last nit, rebased master. Green now.

**Pull request merged**